### ISSUE TITLE
Doesn't detect global .eslintrc.json file

### ISSUE THREAD

**Issue by bfelbo** [BadgeLabs/mocha-eslint#67]
Our file directory is the following:
```
.eslintrc.json
test/0_check_eslint.js
test/1_my_tests.js
```
where `0_check_eslint.js` contains the following:
```
const eslint = require("mocha-eslint")

describe("Check source and test files", function() {
  describe("Should have no eslint errors", function() {
    // Issues are potentially fixable by running `npm run eslint-format`
    eslint(["./src", "./test"],
      { alwaysWarn: false, formatter: "compact", strict: true })
  })
})
```

This doesn't work as the eslint test doesn't check the rules in the `.eslintrc.json`. What are we doing wrong? Do we need to pass the path to the `.eslintrc.json` somehow?
